src/wp-includes/script-loader.php

```diff
@@ -275,6 +279,18 @@ function wp_default_packages_scripts( $scripts ) {
 	 *     'annotations.js' => array('dependencies' => array(...), 'version' => '...'),
 	 *     'api-fetch.js' => array(...
 	 */
+	$assets_path = ABSPATH . WPINC . "/assets/script-loader-packages{$suffix}.php";
+	if ( '' === $suffix && ! file_exists( $assets_path ) ) {
+		/*
+		 * Most likely due to the use of the `script-loader-packages.php` file.
+		 *
+		 * The `script-loader-packages.min.php` file is committed to WordPress
+		 * so use that for the assets instead. The suffix can remain unchanged
+		 * as the JavaScript files themselves are only ever present in both
+		 * their compressed and uncompressed forms.
+		 */
+		$assets_path = ABSPATH . WPINC . '/assets/script-loader-packages.min.php';
+	}
 	$assets = include ABSPATH . WPINC . "/assets/script-loader-packages{$suffix}.php";
```

Comment:
```suggestion
	$assets = include $asset_path;
```
This is seem to have been missed.